new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in the feature1 branch
```

Comment:
Please change `feature1` to `feature6`

Reply:
Hi Mike, that doesn't seem right, since it was added by you in the feature1 branch,

Reply:
> Hi Mike, that doesn't seem right, since it was added by you in the feature1 branch,

Oops, I guess I inadvertently based the feature6 branch on your feature1 branch! Please ignore my comment. I will rebase the to master.